test/integration/scheduler_test.go

```diff
@@ -404,3 +404,177 @@ func makeNPods(c client.Interface, N int) {
 	}
 	wg.Wait()
 }
+
+func TestMultiScheduler(t *testing.T) {
+	framework.DeleteAllEtcdKeys()
```

Comment:
What does it mean by "combinatory ones"? With two scheduler annotations? No, I don't think we support this.